tools/firtool/firtool.cpp

```diff
@@ -90,12 +90,13 @@ static cl::opt<bool>
                       cl::init(false), cl::Hidden, cl::cat(mainCategory));
 
 static cl::opt<bool> disableOptimization("disable-opt",
-                                         cl::desc("disable optimizations"),
+                                         cl::desc("Disable optimizations"),
                                          cl::cat(mainCategory));
 
-static cl::opt<bool> inliner("inline",
-                             cl::desc("Run the FIRRTL module inliner"),
-                             cl::init(true), cl::cat(mainCategory));
+static cl::opt<bool> disableInliner("disable-inliner",
+                                    cl::desc("Disable the module inliner"),
```

Comment:
```suggestion
                                    cl::desc("Disable the inliner pass"),
```
(or thereabouts?)
Nits for sure, but 1)other options end with " pass" and 2)since the option is simply `-disable-inliner` it seems fine to just say this disables the inliner, but don't feel strongly about it if that doesn't seem clear.  Not sure re:capitalization.

Reply:
I went with "Inliner"